api-conta-bancaria/src/main/java/com/apicontabancaria/domain/service/ClienteService.java

```diff
@@ -0,0 +1,26 @@
+package com.apicontabancaria.domain.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.apicontabancaria.domain.model.Cliente;
+import com.apicontabancaria.domain.model.Conta;
+import com.apicontabancaria.domain.model.StatusConta;
+import com.apicontabancaria.domain.repository.ClienteRepository;
+import com.apicontabancaria.domain.repository.ContaRepository;
+import com.apicontabancaria.exceptionhandler.NegocioException;
+
+@Service
+public class ClienteService {
+	
+	@Autowired
+	private ClienteRepository clienteRepository;
+	
+	public Cliente criarCliente(Cliente cliente){
+		Cliente clienteExistente = clienteRepository.findByCpf(cliente.getCpf());
+		if(cliente.getCpf().equals(clienteExistente)) {
+			throw new NegocioException("Cliente já cadastrado...");
```

Comment:
Não curto esse lance de lançar uma exceção generica. Cada exceção deveria ser para uma caso de diferente, e cada uma com o status dela. 404 not foud, 400 badrequest...